Frameworks/CoreText/CTFrame.mm

```diff
@@ -55,9 +56,10 @@ CFRange CTFrameGetVisibleStringRange(CTFrameRef frame) {
     _CTFrame* framePtr = static_cast<_CTFrame*>(frame);
     CFIndex count = 0;
     if (framePtr) {
-        for (CFIndex i = 0; i < framePtr->_lineOrigins.size() && framePtr->_lineOrigins[i].y > 0; ++i) {
-            // Lines are ordered vertically so we can stop once we find one outside of the frame
-            count += static_cast<_CTLine*>([framePtr->_lines objectAtIndex:i])->_strRange.length;
+        for (_CTLine* line in [framePtr->_lines objectEnumerator]) {
```

Comment:
no, _lines is a StrongId and can't be used as a fast enumerable directly, and I prefer this to pretending it's just an NSArray*

---
In reply to: [79682448](https://github.com/Microsoft/WinObjC/pull/989#discussion_r79682448) [](ancestors = 79682448)